main.go

```diff
@@ -78,6 +84,12 @@ func parseFlags(program string, args []string) (c *config, output string, err er
 	if err != nil {
 		return nil, buf.String(), err
 	}
+
+	if conf.gf == "" {
+		flags.PrintDefaults()
+		return nil, buf.String(), errors.New("-gf is required")
+	}
+
```

Comment:
It would seem reasonable to also check for `conf.db` being blank here as the user could do a `-db ""`